app/queries/create_course_export_mutator.rb

```diff
@@ -30,6 +31,7 @@ def create_course_export
         course: course,
         user: current_user,
         reviewed_only: !!reviewed_only,
+        include_inactive_students: !!include_inactive,
```

Comment:
You could safely remove the double not-operator in this case as the value for `include_inactive` will always be one of `True | False`